### DISCUSSION TITLE
New Server|

### DISCUSSION THREAD

**Discussion by JoSoG89** [Ylianst/MeshCentral#6165] [Q&A] [answered]
Hi.

I wanna know if is posible have a group like catchall for the agents without group.

Situation:
I have troubles with my old mesh server, i create a new one starting from 0 at the same domain but all the old disp. cant connect to the new server.

![image](https://github.com/Ylianst/MeshCentral/assets/106213922/a75dfe89-b2b0-45e8-8666-e2d01e66e3f3)
![image](https://github.com/Ylianst/MeshCentral/assets/106213922/026818d6-7b5f-4e22-b4df-509f75eb78bd)

**Comment by si458** [comment, marked as answer]
Check https://github.com/Ylianst/MeshCentral/blob/master/meshcentral-config-schema.json for all the possible options

You can set `orphanAgentUser` which does exactly what u want

https://github.com/Ylianst/MeshCentral/issues/1657#issuecomment-665177939

**Reply by JoSoG89** [reply]
You're true, i was try that but was in wrong position.

Thanks.!
